README.md

```diff
@@ -11,36 +11,71 @@ This repo is split into two parts:
 - Ansible playbooks for integrating with OpenStack services, and creating 
   software middleware platforms on top of ALaSKA infrastructure.
 
+Installation
+------------
+
+Inside a Python 2.7+ virtual environment,
+
+```
+sudo yum install python-virtualenv libselinux-python
+virtualenv --system-site-packages ~/ansible-venv
+source ~/ansible-venv/bin/activate
+pip install -r requirements.txt
+```
+
+Then, download and deploy the role from Ansible Galaxy:
+
+```
+(pushd ansible && ansible-galaxy install -r requirements.yml -p $PWD/roles) || popd
```

Comment:
popd not required as it's in a subshell

Reply:
popd to go back to when the command was executed from otherwise it leaves you inside ./ansible folder afaik

Reply:
more to the point, neither pushd nor popd required - just use cd once in the subshell

Reply:
```
mark@mark-xps15:~$ (pushd src)
~/src ~
mark@mark-xps15:~$ pwd
/home/mark
```